tests/unit/render/TerraformRenderer.spec.js

```diff
@@ -338,12 +338,12 @@ describe('Test TerraformRenderer', () => {
         expect(new TerraformRender(metadata.pluginData).renderFiles()).toEqual([input]);
       });
 
-      it('Should render main file', () => {
+      it.only('Should render main file', () => {
```

Comment:
![image](https://github.com/ditrit/terrator-plugin/assets/6179956/1df82624-348c-4609-ab09-19e67d0280e6)

Reply:
https://youtu.be/Ao3vOCDZd-Y?t=6